source/adapters/cuda/enqueue.cpp

```diff
@@ -242,18 +242,18 @@ setKernelParams(const ur_context_handle_t Context,
           return UR_RESULT_SUCCESS;
         };
 
-        size_t KernelLocalWorkGroupSize = 0;
+        size_t KernelLocalWorkGroupSize = 1;
         for (size_t Dim = 0; Dim < WorkDim; Dim++) {
           auto Err = IsValid(Dim);
           if (Err != UR_RESULT_SUCCESS)
             return Err;
           // If no error then sum the total local work size per dim.
```

Comment:
Can we update the comment to say product here as well, please?

Reply:
Good spot, updated!